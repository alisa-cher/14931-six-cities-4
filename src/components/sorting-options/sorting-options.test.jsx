import React from 'react';
import renderer from 'react-test-renderer';
import SortingOptions from "./sorting-options.jsx";

it(`SortingOptions renders ok`, () => {
  const tree = renderer
    .create(
        <SortingOptions
          onSortingClick={() => {}}
          activeSorting={`to-high`}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
