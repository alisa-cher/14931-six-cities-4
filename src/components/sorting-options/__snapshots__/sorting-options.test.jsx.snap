// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortingOptions renders ok 1`] = `
<form
  action="#"
  className="places__sorting"
  method="get"
>
  <span
    className="places__sorting-caption"
  >
    Sort by
  </span>
  <select
    className="places__sorting-type"
    id="places-sorting"
    onChange={[Function]}
  >
    <option
      className="places__option"
      defaultValue="popular"
      value="popular"
    >
      Popular
    </option>
    <option
      className="places__option"
      defaultValue="popular"
      value="to-high"
    >
      Price: low to high
    </option>
    <option
      className="places__option"
      defaultValue="popular"
      value="to-low"
    >
      Price: high to low
    </option>
    <option
      className="places__option"
      defaultValue="popular"
      value="top-rated"
    >
      Price: high to low
    </option>
  </select>
</form>
`;
